### DISCUSSION TITLE
Radiowecker V4.0.0

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#73] [Ankündigungen / Announcements]
Diskussion start

**Comment by fraeggle** [comment]
Saubere Arbeit.....  Einzig was ich nicht find / oder vielleicht brauch ich nur eine Sehhilfe... Was die Icons bedeuten.  Wolken-->klaro
Aber heute ..... ich kanns auch nicht beschreiben... Rechteck mit waagrechten linien  Nebel?  ;-)

**Reply by beabel** [reply]
Ja, das soll Nebel darstellen. Wenn jemand schönere hat gerne her damit, dann wird es integriert. Ich mache die Tage mal eine Liste mit den Wettercodes und momentanen Icons.

**Reply by beabel** [reply]
Hier ist die Liste der Wettercodes:
  
  ### WMO Wetterinterpretationscodes (WW)

  | Code       | Beschreibung                                   |
  |------------|-----------------------------------------------|
  | 0          | Klarer Himmel                                 |
  | 1, 2, 3    | Überwiegend klar, teilweise bewölkt und bedeckt |
  | 45, 48     | Nebel und Raureifnebel                        |
  | 51, 53, 55 | Nieselregen: Leichte, mäßige und dichte Intensität |
  | 56, 57     | Gefrierender Nieselregen: Leichte und dichte Intensität |
  | 61, 63, 65 | Regen: Leichte, mäßige und starke Intensität   |
  | 66, 67     | Gefrierender Regen: Leichte und starke Intensität |
  | 71, 73, 75 | Schneefall: Leichte, mäßige und starke Intensität |
  | 77         | Schneegriesel                                 |
  | 80, 81, 82 | Regenschauer: Leichte, mäßige und heftige     |
  | 85, 86     | Schneeschauer: Leicht und stark               |
  | 95 *       | Gewitter: Leicht oder mäßig                   |
  | 96, 99 *   | Gewitter mit leichtem und starkem Hagel       |

Hier die aktuellen Icons: (Achtung nicht im Dark Mode erkennbar ;-)
Die Nummer im Dateinamen ist der Wettercode, bei welchem er angezeigt wird.

![0_clear](https://github.com/user-attachments/assets/67ccc72c-ace9-401e-ad21-03cb3d1243cd)
![1_mostly-clear](https://github.com/user-attachments/assets/2db96ce2-c5b5-4457-80ad-a9a9462985c4)
![2_partly-cloudy](https://github.com/user-attachments/assets/e1a8b20a-6572-4423-85ec-ef4b42aa8b28)
![3_overcast-cloudy](https://github.com/user-attachments/assets/cd282d7d-0bcb-41a3-856f-b4cf0b227503)
![45_48_fog](https://github.com/user-attachments/assets/b274fe75-f705-4d0a-91fe-aaf99f79fb54)
![51_53_55_80_drizzle](https://github.com/user-attachments/assets/d6ee06a0-2e1f-4494-924f-c4cd6a349111)
![56_57_freezingdrizzle](https://github.com/user-attachments/assets/e4aa27f2-aacc-4b0a-bcd4-dff1ae05385f)
![66_67_sleet](https://github.com/user-attachments/assets/d970a36c-2e6b-43cd-aecc-c283f3662a0d)
![77_85_86_71_73_75_snow](https://github.com/user-attachments/assets/41386dce-b71a-46cf-ac8b-560c3f0505b2)
![81_82_61_63_65_rain](https://github.com/user-attachments/assets/59cd0d77-5207-495f-8b70-58a4dd433895)
![95_lightning](https://github.com/user-attachments/assets/3a608861-91c2-4e6e-915d-f3fe8477c83d)
![96_99_thunderstorm](https://github.com/user-attachments/assets/02d0f861-a972-45a1-9bde-5f77b00d134a)

**Comment by ziva49** [comment]
Hallo, frohes neues Jahr 2025
So funktionieren die Adressen französischer Radiostreams ohne Hashing. Ich danke Ihnen für Ihre Hilfe!
bis bald.
Beispielablauf 👍 http://icecast.rtl.fr/rtl-1-44-64?listen=webCwsBCggNCQgLDQUGBAcGBg
